### PR TITLE
Update 214

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
- update Binder link
- Delete code comments
- add benchmark app test

### CODE REVIEW

**Review by paularamo — APPROVED** [review]

File: notebooks/214-vision-paddle-classification/214-vision-paddle-classification.ipynb [hunk 1]
```diff
@@ -320,7 +318,41 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89af7e9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 'latency': device performance mode will be set to LATENCY.\n",
```
Comment (ryanloney) [inline notebooks/214-vision-paddle-classification/214-vision-paddle-classification.ipynb:343]:
```suggestion
    "# 'latency': device performance optimized for LATENCY.\n",
```

**Review by ryanloney — CHANGES_REQUESTED** [review]

**Review by ryanloney — APPROVED** [review]

**Comment by ryanloney** [comment]
@Debskij please review and approve when you can

**Comment by adrianboguszewski** [comment]
The link in notebooks/README is still broken!

**Comment by OpenVINO-dev-contest** [comment]
> The link in notebooks/README is still broken!

fixed ,thanks reminder

**Pull request merged**